docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Running Teleport with self-signed certs
+description: How run Teleport using self-certs
+---
+
+In this guide, we will show you how to run Teleport with self-signed certs, with remedies for problems caused
+by using self-signed certs.
+
+The Teleport Proxy Service authenticates itself to clients via TLS x509 certificates.
+If certificates are not configured for the Proxy Service, then it uses self-signed certificates,
+which clients will not trust by default. This can cause problems when using tools like
```

Comment:
All of these tools are used to interact with a Teleport cluster.
```suggestion
which clients will not trust by default. This can cause problems interacting with your Teleport cluster via Teleport cli tools, such as
```